orthoinference/src/main/java/org/reactome/orthoinference/Main.java

```diff
@@ -25,10 +24,15 @@ public static void main(String[] args) throws Exception {
 			logger.fatal("Please include a 4-letter species code as the first argument (eg: mmus)");
 			System.exit(0);
 		}
-		
+
 		Properties props = new Properties();
 		props.load(new FileInputStream(pathToConfig));
-        EventsInferrer.inferEvents(props, pathToConfig, speciesCode);
+        EventsInferrer.inferEvents(props, speciesCode);
+
+        // Link report file to report_ortho_inference.txt in website_files_update directory
+        String pathtoWebsiteFilesUpdateWithReportFile = props.getProperty("pathToWebsiteFilesUpdateFolder") + "report_ortho_inference.txt";
+        String reportFilename = "report_ortho_inference_test_reactome_" + props.getProperty("releaseNumber") + ".txt";
+        Runtime.getRuntime().exec("ln " + reportFilename + " " + pathtoWebsiteFilesUpdateWithReportFile);
```

Comment:
It is possible to create file links from within Java: https://docs.oracle.com/javase/tutorial/essential/io/links.html#symLink